src/WebUI/Pods/PodsLeftPanel.scss

```diff
@@ -1,5 +1,17 @@
 @import "../Common/Common.scss";
 
 .pods-left-panel-container {
-    margin-top: 60px;
+    margin-top: 15px;
+
+    .pod-left-panel-header {
+        padding-left: 20px;
+    }
+
+    .pod-left-panel-table-header {
+        padding-left: 20px;
+        margin-top: 30px;
+        margin-bottom: 30px;
+        font-size: 14px;
+        font-weight: 600;
```

Comment:
we should see if we have a better way for this? 600

Reply:
using fontWeightMedium wherever this was required